src/game-of-life/publishing-to-npm.md

```diff
@@ -0,0 +1,57 @@
+# Publishing to npm
+
+Now that we have a working, fast, *and* small `wasm-game-of-life` package, we
+can publish it to npm so other JavaScript developers can reuse it, if they ever
+need an off-the-shelf Game of Life implementation.
+
+## Prerequisites
+
+First, [make sure you have an npm account](https://www.npmjs.com/signup).
+
+Second, make sure you are logged into your account locally, by running this
+command:
+
+```
+wasm-pack login
+```
+
+## Publishing
+
+Make sure that the `wasm-game-of-life/pkg` build is up to date by running
+`wasm-pack` inside the `wasm-game-of-life` directory:
+
+```
+wasm-pack init
+```
+
+Take a moment to check out the contents of `wasm-game-of-life/pkg` now, this is
+what we are publishing to npm in the next step!
+
+When you're ready, run `wasm-pack publish` to upload the package to npm:
+
+```
+wasm-pack publish
+```
+
+That's all it takes to publish to npm!
+
+...except other folks have also done this tutorial, and therefore the
```

Comment:
We have a `--scope=` flag for this purpose so:

`wasm-pack publish --scope="mgattozzi"`

and I'd recommend using that

Reply:
That only works if you pay npm for private packages, since we don't expose `--access public` or whatever npm flags yet.

Once we do, then we can definitely update this.